cli-signer/Cargo.toml

```diff
@@ -1,10 +1,11 @@
 [package]
-authors = ["Parity <admin@parity.io>"]
+description = "Parity Ethereum CLI Signer Tool"
 description = "Parity Cli Tool"
```

Comment:
Duplicate description causing build errors
```suggestion
```

Reply:
Sorry, didn't realise i could have just commited your suggestion. I've pushed a commit removing the duplicate